validators/helper.go

```diff
@@ -0,0 +1,107 @@
+package validators
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+func NewValidatorFromType(t ValidatorType) Validator {
+	switch t {
+	case ECDSAValidatorType:
+		return new(ECDSAValidator)
+	case BLSValidatorType:
+		return new(BLSValidator)
+	}
+
+	return nil
+}
+
+func NewValidatorSetFromType(t ValidatorType) Validators {
+	switch t {
+	case ECDSAValidatorType:
+		return new(ECDSAValidators)
+	case BLSValidatorType:
+		return new(BLSValidators)
+	}
+
+	return nil
+}
+
+func ParseValidator(t ValidatorType, s string) (Validator, error) {
+	switch t {
+	case ECDSAValidatorType:
+		return ParseECDSAValidator(s)
+	case BLSValidatorType:
+		return ParseBLSValidator(s)
+	default:
+		return nil, fmt.Errorf("invalid validator type: %s", t)
+	}
+}
+
+func AddressesToECDSAValidators(addrs ...types.Address) *ECDSAValidators {
+	set := make(ECDSAValidators, len(addrs))
+
+	for idx, addr := range addrs {
+		set[idx] = &ECDSAValidator{
+			Address: addr,
+		}
+	}
+
+	return &set
+}
+
+func ParseECDSAValidators(ss []string) (*ECDSAValidators, error) {
+	set := make(ECDSAValidators, len(ss))
+
+	for idx, s := range ss {
+		val, err := ParseECDSAValidator(s)
+		if err != nil {
+			return nil, err
+		}
+
+		set[idx] = val
+	}
+
+	return &set, nil
+}
+
+func ParseBLSValidator(s string) (*BLSValidator, error) {
+	subValues := strings.Split(s, ":")
+
+	if len(subValues) != 2 {
+		return nil, fmt.Errorf("invalid validator format, expected [Validator Address]:[BLS Public Key]")
```

Comment:
Error should be standardized

Reply:
I'm afraid I don't see what you said. Would you kindly tell me your suggestions?
@zivkovicmilos

Reply:
Why are you using `fmt.Errof` instead of `errors.New`?

You're not formatting anything

Reply:
@zivkovicmilos I'm sure the code at latest commit doesn't have this part.